### DISCUSSION TITLE
Problem to add remotely a client to a vpn Server.

### DISCUSSION THREAD

**Discussion by Gh21VPN** [SoftEtherVPN/SoftEtherVPN#1847] [General]
Hi, there!

Thank you for using SoftEther.

If you are running SoftEther VPN 4.x (i.e. Stable Edition), please read the comparison with Developer Edition at: 

https://github.com/SoftEtherVPN/SoftEtherVPN#comparison-with-stable-edition

Before you submit an issue, please read the following:

Is this a question?

- If the answer is "yes", then please ask your question on [www.vpnusers.com](http://www.vpnusers.com).
The issue section on GitHub is reserved for bugs and feature requests.

- If the answer is "no", please read the following:

We provide a template which is specifically made for bug reports, in order to be sure that the report includes enough details to be helpful.

Please use or adapt it as needed.

---

### Prerequisites

* [ ] Can you reproduce?
* [x] Are you running the latest version of SoftEtherVPN?

**SoftEther version:**
**Component:** [Server, Client, Bridge, etc.]
**Operating system:** [Windows, Linux, BSD, macOS, etc.]
**Architecture:** [64 bit, 32 bit]

[In case it's a computer with known specs, such as the Raspberry Pi, you can specify it omitting the details.]
**Processor:** [Specify brand and model. Example: AMD Ryzen 7 1800x]

### Description

[Description of the bug]

**Expected behavior:**
[What you expected to happen]

**Actual behavior:**
[What actually happened]

### Steps to reproduce

1. [First step]
2. [Second step]
3. [And so on...]

**Comment by shakibamoshiri** [comment]
@Gh21VPN 
what is the issue ? Both "vpncmd" and RPC-JSON can add a new users .
